types/base64util/index.d.ts

```diff
@@ -0,0 +1,33 @@
+// Type definitions for base64util 2.2
+// Project: https://github.com/duzun/base64util
+// Definitions by: Jonathan Ehman <https://github.com/jpehman>
```

Comment:
Is this your personal account? Can you confirm this is the github account you'll be using for e.g. create/review future PRs to this package?

Reply:
Yes @jpehman is my personal account. I will use it to contribute and update this package in the future.